lecture_4/pmb2164/pmb2164.tex

```diff
@@ -0,0 +1,53 @@
+
+% use a sans serif font
+\renewcommand{\familydefault}{\sfdefault}
+
+\begin{document}
```

Comment:
remove \begin{document} and \maketitle as well. they're included in template.tex too.